tsdb/engine/tsm1/engine.go

```diff
@@ -2226,7 +2240,7 @@ func (e *Engine) compactHiPriorityLevel(grp CompactionGroup, level int, fast boo
 
 // compactLoPriorityLevel kicks off compactions using the lo priority policy. It returns
 // the plans that were not able to be started
-func (e *Engine) compactLoPriorityLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup) bool {
+func (e *Engine) compactLoPriorityLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup, pointsPerBlock int) bool {
```

Comment:
I like passing `pointsPerBlock` in, but I'd like to see `pointsPerBlock` after `fast`. That way the parameters that impact how the compaction is performed are together.